modules/reports/views/histogram/options.php

```diff
@@ -90,22 +90,12 @@
 		</td>
 	</tr>
 	<tr>
-		<td>
-			<br />
-			<?php echo help::render('skin') ?>
-			<label for="skin" id="skin_lbl"><?php echo _('Skin') ?></label>
-		</td>
-		<td></td>
 		<td>
 			<?php echo help::render('description') ?>
 			<label for="description" id="descr_lbl"><?php echo _('Description') ?></label>
 		</td>
 	</tr>
 	<tr>
-		<td style="vertical-align: top;">
-			<?php echo form::dropdown(array('name' => 'skin'), ninja::get_skins(), $options['skin']); ?>
-		</td>
-		<td></td>
```

Comment:
It's probably a non-issue but it may be safer to keep the number of columns in the table consistent? It does make a small difference to how the row is highlighted when hovered over. Up to you.